templates/environment/partials/environment-manager-role.yml

```diff
@@ -224,3 +225,12 @@ EnvironmentManagerRole:
             - 'cloudformation:DeleteStack'
           Resource:
             - !Sub 'arn:aws:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/${AWS::StackName}/*'
+        - Sid: AdministerTask
+          Effect: Allow
+          Action: [
+            "ecr:BatchDeleteImage"
+          ]
+          Resource: "*"
+          Condition: {"ForAllValues:StringEquals": {"aws:TagKeys": "copilot-task"}}
```

Comment:
Are we using this in this PR?

Reply:
Oh, no, that snuck in from another branch I based this on. Thanks for the check.